packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,170 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+Visual Design Comments:
+
+Brandon's Figma spec has designs for two accordion item heights, 24px and 32px. In previous cases like the nimble button, we've only implemented one height (in that case only the 32px button was implemented). Do we want to continue doing this, and if so, which height should be implemented?
+
+Brandon's Figma spec also only shows designs for one accordion item at a time. How should accordion items in each appearance type be separated from each other (ex. a line between accordion items, transparent space between accordion items, etc.)?
+
+The Figma design does not have a design for the "error" focusVisible state. This might be confusing, as keyboard users who tab to the error accordion will suddenly see the red border and exclamation mark icon disappear in the focusVisible state.
+
+When should each `appearance` type be used? Will there be specific guidance on this, and if so, will it be similar to the guidance for other Nimble components that use the same `appearance` types (ex. the Nimble-Button)?
+
+## Design
+
+Accordion group, appearance ghost with an error-visible accordion item
+
+```
+<nimble-accordion appearance="ghost">
+    <nimble-accordion-item>
+        <span slot="heading">Accordion one</span>
+            <div slot="default">
```

Comment:
I'm assuming this `div` is going in the default slot, not a slot with the name `"default"`? If so, you don't need to specify `slot="default"`. Simply having the content be a child of the `nimble-accordion-item` with not slot specified will do what you want.

Also, since this `div` isn't a child of the `span` on the line above, it should be out-dented once.

Same for the other instances below.